src/benchmark/md5.c

```diff
@@ -0,0 +1,10 @@
+#include <openssl/md5.h>
```

Comment:
Given that MD5 is not exactly used anymore, how about comparing with SHA1/SHA256? (I know that blake is supposed to compete with the speed of MD5, so this test is cool, but another test to compare performance with algorithms that are actually used would be great.)